Entity/StandardMarker.MiniMap.cs

```diff
@@ -16,8 +16,13 @@ public partial class StandardMarker {
         public override RectangleF? RenderToMiniMap(SpriteBatch spriteBatch, Rectangle bounds, (double X, double Y) offsets, double scale, float opacity) {
             if (IsFiltered(EntityRenderTarget.Map) || this.Texture == null) return null;
 
-            if ((!this.MapVisibility     || !_packState.UserConfiguration.MapShowMarkersOnFullscreen.Value) && GameService.Gw2Mumble.UI.IsMapOpen) return null;
-            if ((!this.MiniMapVisibility || !_packState.UserConfiguration.MapShowMarkersOnCompass.Value)    && !GameService.Gw2Mumble.UI.IsMapOpen) return null;
+            bool isMapOpen = GameService.Gw2Mumble.UI.IsMapOpen;
+
+            var mapShowMarkersOnFullscreen = _packState.UserConfiguration.MapShowMarkersOnFullscreen.Value;
+            if (mapShowMarkersOnFullscreen != VisibilityLevel.Always && (!this.MapVisibility || mapShowMarkersOnFullscreen == VisibilityLevel.Never) && isMapOpen) return null;
+
+            var mapShowMarkersOnCompass = _packState.UserConfiguration.MapShowMarkersOnCompass.Value;
+            if (mapShowMarkersOnCompass != VisibilityLevel.Always && (!this.MiniMapVisibility || mapShowMarkersOnCompass == VisibilityLevel.Never) && !isMapOpen) return null;
```

Comment:
This section here is growing in complexity.  I'm willing to merge as is, but could you add a `// TODO:` to have it split out some added just above it?

Reply:
I extracted one of the conditions into a variable, and I changed the order of the checks in order to make it a bit more readable. Let me know what you think.